charts/linkerd-control-plane/values.yaml

```diff
@@ -286,10 +289,16 @@ proxy:
     collector:
       # -- The collector endpoint to send traces to.
       endpoint: ""
-      # -- The identity of the collector in the linkerd mesh. If the collector
-      # is unmeshed, this should remain unset.
+      # -- The identity of the collector in the linkerd mesh.
       meshIdentity:
+        # -- Service account name for the trace collector. If there's no explicitly set service account for the trace
+        # collector, this should be set to the name of the deployment/statefulset/daemonset of the trace collector
+        # instead.
```

Comment:
is this true? if the collector doesn't specify a service account, I thought it used the default service account for it's namespace.